examples/shipping/booking/endpoint.go

```diff
@@ -0,0 +1,138 @@
+package booking
+
+import (
+	"time"
+
+	"github.com/go-kit/kit/endpoint"
+	"github.com/go-kit/kit/examples/shipping/cargo"
+	"github.com/go-kit/kit/examples/shipping/location"
+	"golang.org/x/net/context"
```

Comment:
Please move third-party imports to their own whitespace-separated block, after stdlib and above go-kit. (Further comments of this type elided.)